Directory.Build.props

```diff
@@ -50,11 +50,6 @@
     <SentryUploadSources>true</SentryUploadSources>
   </PropertyGroup>
 
-  <!-- If running in CI and auth token is not set, disable SentryCLI completely (to avoid logging warnings). -->
-  <PropertyGroup Condition="'$(GITHUB_ACTIONS)' == 'true' And '$(SENTRY_AUTH_TOKEN)' == ''">
-    <UseSentryCLI>false</UseSentryCLI>
-  </PropertyGroup>
```

Comment:
This (and other similar cases below) is now handled by a condition in `Sentry.targets` property `UseSentryCLI`